week4/week4-079.NumberStatistics/src/Main.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class Main {
+    
+    public static void main(String[] args) {
+        Scanner reader = new Scanner(System.in);
+        
+        NumberStatistics stats = new NumberStatistics();
+        NumberStatistics even = new NumberStatistics();
+        NumberStatistics odd = new NumberStatistics();
+        
+        System.out.println("Type numbers:");
```

Comment:
Može bez while(true) i break-a opet. Break je inače, kao što sam spomenuo, prilično loša navika.

```
int number = reader.nextInt();
while(number != -1){

 stats.addNumber(number);
 if(number % 2 == 0){
       even.addNumber(number);
} else {
      odd.addNumber(number);
}
number = reader.nextInt();
}
```